shell/browser/api/atom_api_browser_window.cc

```diff
@@ -295,11 +295,11 @@ void BrowserWindow::OnWindowLeaveFullScreen() {
 #endif
 }
 
-void BrowserWindow::Focus() {
+void BrowserWindow::Focus(const gin_helper::Dictionary& options) {
```

Comment:
Because the `options` object is optional you can use `base::Optional`

```suggestion
void BrowserWindow::Focus(const base::Optional<gin_helper::Dictionary>& options) {
```

This means you don't have to do the extra bit of JS 👍

Reply:
Ah, cool! I wasn't aware of `baser::Optional` - I'll get that integrated once I've fixed my local build (🤦‍♂) and update the diff.